ipywidgets/widgets/widget.py

```diff
@@ -112,6 +112,7 @@ class Widget(LoggingConfigurable):
     # Class attributes
     #-------------------------------------------------------------------------
     _widget_construction_callback = None
+    _read_only_enabled = True
```

Comment:
Hmm, what's the argument for this?  Seems redundant to me, like the `readonly` kwargs on the traits should be enough, no?